Cargo.toml

```diff
@@ -26,6 +26,7 @@ ratatui = { version = "0.27", features = [
         "unstable-rendered-line-info",
         "unstable-widget-ref",
 ] }
+rpassword = "7.3.1"
```

Comment:
I found this dependency via StackOverflow. This ensures that the password isn't shown when the user enters it. There's probably a way to do this with the standard library, but this is pretty much my first time working with Rust, so I wouldn't know how.